mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -1505,6 +1505,7 @@ func ResourceContainerCluster() *schema.Resource {
 				Description: `Node pool configs that apply to all auto-provisioned node pools in autopilot clusters and node auto-provisioning enabled clusters.`,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"node_kubelet_config": schemaKubeletConfig(),
```

Comment:
Ok, so revert the change breaking out `schemaKubeletConfig` and then create a new schema (`schemaNodePoolAutoConfigNodeKubeletConfig`? in `node_config.go.erb`? -- I think `schemaNodeKubeletConfig` would be potentially confusing since `nodeKubeletConfig` is what `kubeletConfig` is in some places) that just includes `insecure_kubelet_readonly_port_enabled`, right?

Reply:
Ok, updated, pushed the changes so far, and running the test locally as well.